document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/AccountsHandlerImplTest.java

```diff
@@ -67,9 +72,20 @@ void testGetAbridgedAccountsDataFailureFromServiceLayer() throws ServiceExceptio
     void testGetAbridgedAccountsData() throws ServiceException, HandlerException {
         when(accountsService.getAccounts(anyString(), anyString())).thenReturn(createAccountsObject());
         when(accountsService.getAbridgedAccounts(anyString(), anyString())).thenReturn(createCurrentPeriodAbridgedAccountObject());
+        when(companyService.getCompanyProfile(anyString())).thenReturn(createCompanyProfile());
+        when(transaction.getCompanyNumber()).thenReturn("000667733");
```

Comment:
Can we replace string literals with constants please, in case they're reused later